config.js

```diff
@@ -57,16 +57,27 @@ mapkey('p', "Open the clipboard's URL in the current tab", () => { Clipboard.rea
 // Open Clipboard URL in new tab
 map('P', 'cc');
 
-// Current tab Omnibar
+// Open a URL in current tab
 map('o', 'go');
 
+// Choose a buffer/tab
+map('b', 'T');
+
+// Edit current URL, and open in same tab
+map('O', ';U');
+
+// Edit current URL, and open in new tab
+map('T', ';u');
+
 // History Back/Forward
 map('H', 'S');
 map('L', 'D');
 
 // Scroll Page Down/Up
 mapkey("<Ctrl-d>", "Scroll down", () => { Normal.scroll("pageDown"); });
-mapkey("<Ctrl-u>", "Scroll down", () => { Normal.scroll("pageUp"); });
+mapkey("<Ctrl-u>", "Scroll up", () => { Normal.scroll("pageUp"); });
+map('<Ctrl-b>', 'U');  // scroll full page up
+//map('<Ctrl-f>', 'P');  // scroll full page down -- looks like we can't overwrite browser-native binding
```

Comment:
Any idea whether it's possible to make `Ctrl+f` work?

Reply:
Don't think so, it'll have to do.